godtools/App/Share/SwiftUI Views/ActivityIndicator.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ActivityIndicator.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 4/15/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+// TODO: - GT-1533: In iOS 14+, we can use SwiftUI's `ProgressView` to get a built-in spinner and delete this one.
+/// Wraps the UIKit ActivityIndicator to make it useable in SwiftUI.
+struct ActivityIndicator: UIViewRepresentable {
+    
+    typealias UIViewType = UIActivityIndicatorView
```

Comment:
Will do!  Is there a style guide anywhere?  Might be worth adding one to the readme.

Reply:
Hey Rachael we do not have a style guide.  You meaning programming style guide?  We have a Jira task to add in Swift Lint, just haven't gotten to that task yet.

Reply:
Yeah!  To keep track of project-specific practices like files should be sorted alphabetically, only one declaration per file, etc.  Would make it easier when someone onboards.  I know that I, out of habit, put the // MARKs everywhere, but that may not be considered clean either?  SwiftLint could help too.

Reply:
That's a good idea.  I'll go ahead and create a Jira task for that.  It would be good to look into suggestions on using MARKS.  I have some used in places as well.

Reply:
Yeah the "Clean" answer seems to be to avoid using position markers, they're clutter and people mostly ignore them.  I'd tend to disagree, I do use them for navigating through files quickly.  Especially since Xcode pulls them into the minimap and the method dropdown selector.  But on the other hand, if I need help navigating, maybe the file is too big and should be broken up?

Reply:
Hey Rachael I feel like it's either we remove all of them or there should be a consistency when using them.  I'll do a little more research and it would be good to include the final resolution into the style guide.  For now let's just keep as is.